f8a_report/helpers/report_helper.py

```diff
@@ -536,6 +542,12 @@ def normalize_worker_data(self, start_date, end_date, stack_data, worker,
                                                    unique_stacks_with_deps_count,
                                                    avg_response_time,
                                                    unknown_deps_ingestion_report),
+                'golang': self.get_ecosystem_summary('golang', total_stack_requests, all_deps,
```

Comment:
Can the same be done here, instead of adding for each ecosystem.

Reply:
Ack!